app/stac_api/managers.py

```diff
@@ -0,0 +1,163 @@
+import logging
+from datetime import datetime
+
+from django.contrib.gis.db import models
+from django.contrib.gis.geos import Point
+from django.contrib.gis.geos import Polygon
+from django.db.models import Q
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework.exceptions import ValidationError
+
+from stac_api.validators import validate_geometry
+
+logger = logging.getLogger(__name__)
+
+
+class ItemQuerySet(models.QuerySet):
+
+    def by_bbox(self, bbox):
```

Comment:
I'd maybe call it `filter_by_bbox`, slightly more descriptive what the method is doing.

Reply:
If this change is implemented, it might make sense for consistence to also rename the other `by_something` filters, e.g. `by_datetime`.

Reply:
That was actually my idea to be more concise, but I'm also OK with `filter_by_bbox`, like @hansmannj  mentioned we need then to be consistent everywhere.